Library/Homebrew/tap.rb

```diff
@@ -233,7 +233,7 @@ def core_tap?
   def install(options = {})
     require "descriptions"
 
-    full_clone = options.fetch(:full_clone, false)
+    full_clone = options.fetch(:full_clone, true)
```

Comment:
If I understand the code correctly, this is a default value in case the option is not defined at all.
But it is defined always, looking at `cmd/tap.rb`.

Reply:
Good catch, will fix.

Reply:
https://github.com/Homebrew/brew/pull/7004